css/style.css

```diff
@@ -1,19 +1,358 @@
+/* ==========default============ */
 
- /* We are using the 'Roboto' font from Google. This has already been added to your HTML header */
+:root {
+  --clr-orange: #e0633a;
+  --clr-black: #333;
+  --clr-gray: #878787;
+  --clr-white: rgb(255, 255, 255);
+  --clr-light-gray: #aeaeae;
+  --clr-orange-light: #fcefeb;
```

Comment:
Love the use of CSS variables! If you wanted to go the extra mile, you could add one for a "spacing unit" here as well, which you could then use when defining margins/padding on various elements.